packages/core/src/common/message-service-protocol.ts

```diff
@@ -16,23 +16,37 @@
 
 import { injectable, inject } from 'inversify';
 import { ILogger } from './logger';
+import { Event } from '../common/event';
 
 export const messageServicePath = '/services/messageService';
 
 export enum MessageType {
     Error = 1,
     Warning = 2,
     Info = 3,
-    Log = 4
+    Log = 4,
+    Progress = 5
 }
 
-export interface Message {
+export interface MessageArguments {
```

Comment:
hello, should we avoid to rename the class to be backward compliant ?

Reply:
Agree, renamed back.